spec/unit/face/help_spec.rb

```diff
@@ -122,13 +125,6 @@
           expect(Puppet::Face[name, :current]).to have_a_summary
         end
       end
-
-      it "returns an 'unavailable' summary if the face application raises a LoadError" do
```

Comment:
There's a version of this test for legacy applications up a little higher in the file. Are face applications no longer supposed to do this?

Reply:
@jhelwig  Ok, I added context's around the legacy tests at the at new line 190 and we cover subcommand and action failures at line 50 and 69 respectively.
I also converted the double quotes to singles while I was at it.